### PR TITLE
Docker support with Nginx

### PR DESCRIPTION
**Pull request by DanielAzulayy**
A PR for: https://github.com/zuramai/mazer/issues/111

Dockerizing mazer with Nginx support

### CODE REVIEW

**Comment by zuramai** [comment]
I'm trying to run as the steps you wrote in the README, I got this error:

```
Sending build context to Docker daemon  137.2MB
Step 1/8 : FROM node:16 AS frontend
 ---> 954cea825b78
Step 2/8 : WORKDIR /frontend
 ---> Using cache
 ---> 034cafe2cb73
Step 3/8 : COPY package*.json /frontend
When using COPY with more than one source file, the destination must be a directory and end with a /
```

I guess you missing the `/` at the end of the target directory?

```diff
- COPY package*.json /frontend
+ COPY package*.json /frontend/
```

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM node:16 AS frontend
+
+WORKDIR /frontend
+COPY package*.json /frontend
```
Comment (zuramai) [inline Dockerfile:4]: add `/` at the end of the line

**Review by zuramai — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (DanielAzulayy) [reply Dockerfile:4]: No problem
Does it work for you when you added / at the end of line 4?

I'm running Mac and both worked for some reason (with and without /)

Comment (zuramai) [reply Dockerfile:4]: I tried adding `/` at the end of line 4 and it works!

Comment (DanielAzulayy) [reply Dockerfile]: Cool, interesting!

**Review by zuramai — APPROVED** [review]

**Comment by zuramai** [comment]
Great!
Thank you very much for your contribution :smile:

File: Dockerfile [hunk 2]
```diff
@@ -1,11 +1,11 @@
 FROM node:16 AS frontend
 
 WORKDIR /frontend
```
Comment (bagussatoto) [inline Dockerfile:3]: font end ?

Comment (DanielAzulayy) [reply Dockerfile:3]: yeah, why?

**Comment by DanielAzulayy** [comment]
> Great! Thank you very much for your contribution 😄

Thanks for the great UI!

**Pull request merged**